run-validation/flow_validation.py

```diff
@@ -578,6 +578,19 @@ def main(flow_file, charge_only):
             output.savefig()
             plt.close()
 
+        # Check the sums of the fraction field for charge deposition. They should add to 1.
+        if 'mc_truth' in flow_h5.keys():
+            fractions = flow_h5['mc_truth/packet_fraction/data']['fraction']
+            summed_fractions = fractions.sum(axis=-1)
+            fig, ax = plt.subplots(constrained_layout = True)
+            ax.hist(summed_fractions, bins = np.arange(0.895,1.1,0.01), density = True)
```

Comment:
What if something is terribly wrong and the fraction is outside of the hardcoded range?...

Reply:
Yee, I've expanded the range to include everything and changed the y-axis to log scale.

Reply:
What about just specifying the number of bins with `bins = 250` (or whatever) and letting the plotter autorange?